cmd/csi-powerstore/main.go

```diff
@@ -173,24 +178,29 @@ func main() {
 		opentracing.SetGlobalTracer(t)
 		interList = append(interList, grpc_opentracing.UnaryServerInterceptor(grpc_opentracing.WithTracer(t)))
 	}
+	storageProvider := &gocsi.StoragePlugin{
+		Controller:                controllerService,
+		Identity:                  identityService,
+		Node:                      nodeService,
+		Interceptors:              interList,
```

Comment:
interList is not a good name, inter could be expanded into many things; interceptor, interface for example.